google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITBucketTest.java

```diff
@@ -495,6 +498,39 @@ public void testEnableDisableBucketDefaultEventBasedHold()
     }
   }
 
+  @Test
+  public void testCreateBucketWithAutoclass() throws ExecutionException, InterruptedException {
+    String bucketName = bucketFixture.newBucketName();
+    storageFixtureHttp
+        .getInstance()
+        .create(BucketInfo.newBuilder(bucketName)
+            .setAutoclass(Autoclass.newBuilder().setEnabled(true).build())
+            .build());
+    try {
+      Bucket remoteBucket = storageFixtureHttp.getInstance().get(bucketName);
+
+      assertNotNull(remoteBucket.getAutoclass());
+      assertTrue(remoteBucket.getAutoclass().getEnabled());
+      OffsetDateTime time = remoteBucket.getAutoclass().getToggleTime();
+      assertNotNull(time);
+
+      remoteBucket.toBuilder()
+          .setAutoclass(Autoclass.newBuilder().setEnabled(false).build())
+          .build()
+          .update();
+
+      remoteBucket = storageFixtureHttp.getInstance().get(bucketName);
```

Comment:
Ditto here, we should be able to use `storageFixture` here so that we can cross test this on grpc.
```suggestion
      remoteBucket = storageFixture.getInstance().get(bucketName);
```